docs/Contributing/Feature Requests/Feature_requests.md

```diff
@@ -0,0 +1,6 @@
+## Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea
+fits with the scope and aims of the project. It's up to *you* to make a strong
```

Comment:
What are the scope and aims of the project?

Reply:
This is present in the rests of the doc: https://atos.readthedocs.io/en/latest/

Reply:
Can you address this in the text? Perhaps with a link?